### PR TITLE
Created basic customer controller

### PR DESCRIPTION
**Pull request by wasimghazal**

### CODE REVIEW

**Review by dylancoffey99 — COMMENTED** [review]
This looks good so far. We should continue from this and implement the other methods.

**Review by Miracle2001 — APPROVED** [review]

File: system/views/customer_view.py [hunk 1]
```diff
@@ -13,18 +14,18 @@ def __init__(self, root, controller):
...
         # Frame combobox/entries
         product_list = tk.StringVar()
         product_combobox = ttk.Combobox(self.root, width=34, state="readonly",
                                         textvariable=product_list)
         product_combobox["values"] = ("Laptop", "Monitors", "Mouse", "Keyboard")
-        product_combobox.grid(row=0, column=1, pady=10)
+        product_combobox.grid(row=1, column=1, pady=10)
         product_quantity_entry = ttk.Entry(self.root, width=18)
-        product_quantity_entry.grid(row=0, column=3, padx=10, pady=10)
+        product_quantity_entry.grid(row=1, column=3, padx=10, pady=10)
```
Comment (dylancoffey99) [inline system/views/customer_view.py:28]: You need to revert all the row/padding changes that you made when you edited CustomerView.
It seems like you added +1 to all the rows of the widgets. product_label.grid padding should be reverted to 10 also.

File: system/controllers/customer_controller.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import tkinter as tk
+from system import views
+from system.database.db_handler import ProductDB
+from system.controllers.abstract_controllers import AbstractCustomerController
+
+
+class CustomerController(AbstractCustomerController):
+    def __init__(self, root, user, controller):
```
Comment (dylancoffey99) [inline system/controllers/customer_controller.py:8]: Maybe rename "controller" to "access_controller".

File: system/controllers/access_controller.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import hashlib
 import tkinter as tk
 from system import views
+from system.controllers.customer_controller import CustomerController
 from system.database.db_handler import UserDB
 from system.models.users.customer import Customer
 from system.controllers.abstract_controllers import AbstractAccessController
```
Comment (dylancoffey99) [inline system/controllers/access_controller.py:7]: Move the CustomerController import so it's just above AbstractAccessController.

File: system/controllers/customer_controller.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        self.root = root
+        self.user = user
+        self.controller = controller
```
Comment (dylancoffey99) [inline system/controllers/customer_controller.py:11]: Rename here as well:
self.access_controller = access_controller

File: system/controllers/customer_controller.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        self.product_db = ProductDB("system/database/productDB")
+        self.view = views.CustomerView(self.root, self)
+        self.input = {}
+
+    def logout_user(self, root: tk.Tk, frame: tk.Frame):
+        self.destroy_frame(frame)
+        for child in root.winfo_children():
+            child.destroy()
+        self.view = views.LoginView(self.root, self.controller)
```
Comment (dylancoffey99) [inline system/controllers/customer_controller.py:20]: Rename here as well:
self.access_controller

**Review by dylancoffey99 — CHANGES_REQUESTED** [review]

**Review by dylancoffey99 — APPROVED** [review]
This looks good to merge as a basis. Its other methods can be continued in another pull request/branch.

**Pull request merged**